Sources/ArcGISToolkit/Components/Form/FormView.swift

```diff
@@ -0,0 +1,109 @@
+// Copyright 2023 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import FormsPlugin
+import SwiftUI
+
+/// Forms allow users to edit information about GIS features.
+/// - Since: 200.2
+public struct FormView: View {
+    /// Info obtained from the map's JSON which contains the underlying form definition.
+    @State private var mapInfo: MapInfo?
+    
+    /// The attributes of the provided feature.
+    private let attributes: [String : Any]?
+    
+    /// The map containing the underlying form definition.
+    private let map: Map
+    
+    /// Creates a `FormView` with the given map and feature.
+    /// - Parameter map: The map containing the underlying form definition.
+    /// - Parameter feature: The feature to be edited.
+    public init(map: Map, feature: ArcGISFeature) {
+        self.map = map
+        self.attributes = feature.attributes
+    }
+    
+    public var body: some View {
+        ScrollView {
+            VStack(alignment: .leading) {
+                FormHeader(title: formDefinition?.title)
+                Divider()
+                ForEach(formDefinition?.formElements ?? [], id: \.element?.label) { container in
+                    if let element = container.element {
+                        makeElement(element)
+                    }
```

Comment:
Right now, the elements are pretty squashed together. For now, this is OK, but during the validation UI work, we should add spacing/dividers to better separate elements.